src/octree/detail.rs

```diff
@@ -59,21 +60,33 @@ where
 
     pub(in crate::octree) fn clear(&mut self, child_index: usize) {
         debug_assert!(child_index < 8);
-        match &mut self.content {
-            NodeChildrenArray::Children(c) => {
-                c[child_index] = self.default_key.clone();
-                if 8 == c.iter().filter(|e| **e == self.default_key).count() {
-                    self.content = NodeChildrenArray::NoChildren;
-                }
+        if let NodeChildrenArray::Children(c) = &mut self.content {
+            c[child_index] = self.default_key.clone();
+            if 8 == c.iter().filter(|e| **e == self.default_key).count() {
```

Comment:
I think `default_key` could be renamed to something like `no_child_marker_key` so one doesn't need to read the comment on what it actually means for anything to equal "`default_key`"

Reply:
yeah, I agree actually, we just need to find the name for it.. 
I was thinking more like `empty_key_value`